horace_core/sqw/@sqw/split.m

```diff
@@ -31,11 +84,29 @@
 %
 % Evaluate the size of the resulting split to know what subalgorithm to use
 %
-%split_img_size = 3*numel(data.s)*8; % size of resulting split image
-%split_pix_size = w.pix.num_pixels*PixelDataBase.pix_byte_size;
+split_img_size = 3*numel(w.data.s)*8; % size of resulting split image
+
+% set keep_precision to false as filebacked operations here will be
+% performed without change in precision.
+w.pix.keep_precision = true;
```

Comment:
either the comment or the code line is wrong.

Reply:
comment is wrong